ranger/data/scope.sh

```diff
@@ -141,7 +141,7 @@ handle_image() {
         ## Image
         image/*)
             local orientation
-            orientation="$( identify -format '%[EXIF:Orientation]\n' -- "${FILE_PATH}" )"
+            orientation="$( identify -format '%[orientation]\n' -- "${FILE_PATH}" )"
             ## If orientation data is present and the image actually
             ## needs rotating ("1" means no rotation)...
             if [[ -n "$orientation" && "$orientation" != 1 ]]; then
```

Comment:
`orientation` gives a human-readable orientation so this check would have to change, it's never empty nor a digit.